### PR TITLE
add bom

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

/cc @puerco @cpanato

### CODE REVIEW

File: bom.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+
+environment:
+  contents:
+    packages:
+      - git
```
Comment (luhring) [inline bom.yaml:16]: Are we able to remove this section now that https://github.com/chainguard-dev/melange/pull/239 has merged?

Comment (developer-guy) [reply bom.yaml:16]: oh cool, let's remove it! Thanks for keeping me informed about that @luhring 👋

**Review by luhring — DISMISSED** [review]
🎉

**Review by puerco — DISMISSED** [review]
Yay this is awesome, you read my mind @developer-guy !

**Review by dlorenc — APPROVED** [review]

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**